cosmovisor/README.md

```diff
@@ -261,14 +263,23 @@ Create a new key for the validator, then add a genesis account and transaction:
 ./build/simd collect-gentxs
 ```
 
+Start `cosmosvisor`:
+
+```
+cosmovisor run start
+```
```

Comment:
You cannot start cosmovisor before you "prepare cosmovisor" (the next section). This will not work without having set the genesis binary.

Reply:
right, moving it down.